### PR TITLE
Update dependencies to better account for fapolicyd differences between RHEL8 and OL8

### LINKED ISSUE #385: [BUG] AWS CLI Exclusion For `fapolicyd` Malfunctions on OL8
**Describe the bug**

The `aws_cli_v2` state is not (currently) compatible with Oracle Linux 8. While the state is constructed such that it _should_ simply avoid executing these states if the `fapolicyd` RPM is not installed, the states that have explicitly-asserted dependencies still attempt to execute (and fail) causing watchmaker to fail on Oracle Linux 8. 

**Severity**

Completely Broken on non RHEL8 EL8 systems

**To Reproduce**
1. Launch an Oracle Linux 8 system 
2. Execute watchmaker
3. Wait till it attempts to execute the `ash-linux.el8.VendorSTIG.aws_cli_v2` content and fails
4. Check logs for content similar to:

~~~
2023-04-25 15:38:48,863 [watchmaker][CRITICAL][6590]:
Traceback (most recent call last):
  File "/usr/local/bin/watchmaker", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/watchmaker/cli.py", line 115, in main
    sys.exit(watchmaker_client.install())
  File "/usr/local/lib/python3.9/site-packages/watchmaker/__init__.py", line 410, in install
    workers_manager.worker_cadence()
  File "/usr/local/lib/python3.9/site-packages/watchmaker/managers/worker_manager.py", line 64, in worker_cadence
    worker.install()
  File "/usr/local/lib/python3.9/site-packages/watchmaker/workers/salt.py", line 817, in install
    self.process_states(self.salt_states, self.exclude_states)
  File "/usr/local/lib/python3.9/site-packages/watchmaker/workers/salt.py", line 631, in process_states
    raise WatchmakerError(
watchmaker.exceptions.WatchmakerError: Salt state execution failed:
    Exempt AWS CLI v2 From fapolicyd (executable):
        __id__: Exempt AWS CLI v2 From fapolicyd (executable)
        __run_num__: 8
        __sls__: ash-linux.el8.VendorSTIG.aws_cli_v2
        changes: {}
        comment: '/etc/fapolicyd/rules.d/30-aws.rules: file not found'
        duration: 1.86
        name: /etc/fapolicyd/rules.d/30-aws.rules
        result: false
        start_time: '15:35:32.326833'
    Exempt AWS CLI v2 From fapolicyd (sharedlib):
        __id__: Exempt AWS CLI v2 From fapolicyd (sharedlib)
        __run_num__: 9
        __sls__: ash-linux.el8.VendorSTIG.aws_cli_v2
        changes: {}
        comment: '/etc/fapolicyd/rules.d/30-aws.rules: file not found'
        duration: 1.398
        name: /etc/fapolicyd/rules.d/30-aws.rules
        result: false
        start_time: '15:35:32.329820'
    Reload fapolicyd config:
        __run_num__: 10
        __sls__: ash-linux.el8.VendorSTIG.aws_cli_v2
        changes: {}
        comment: 'One or more requisite failed: ash-linux.el8.VendorSTIG.aws_cli_v2.Exempt
            AWS CLI v2 From fapolicyd (executable), ash-linux.el8.VendorSTIG.aws_cli_v2.Exempt
            AWS CLI v2 From fapolicyd (sharedlib)'
        duration: 0.006
        result: false
        start_time: '15:35:32.333786'
~~~

**Expected behavior**

This state should either properly configure the AWS CLI to operate under `fapolicyd` or should skip the attempt and move cleanly onto the remaining formulae

**Deviance Description**

State goes splodey.

**Screenshots**

**Additional context**

Manual steps to take to work around the problem:
1. Install the `fapolicyd` RPM
2. Enable and start the `fapolicyd` service
3. Perform a standalone re-execution of the state
 
Appears to be a confluence of factors:

- RHEL8's and OL8's SCAP content varies with respect to the installation and enablement of the `fapolicyd` RPM and service.
- The Saltstack requisites underpinning the execution of the states is not currently correct

**Fix Suggestions**

Either:
- Update the state to ensure that the `fapolicyd` RPM is always installed _and_ the `fapolicyd` service is running
- Update the state's requisites to function as expected (assuming there's not an underlying SaltStack issue causing the asserted requisites to fail)

### PR DESCRIPTION
**Pull request by ferricoxide**
closes #385

### CODE REVIEW

File: ash-linux/el8/VendorSTIG/aws_cli_v2.sls [hunk 1]
```diff
@@ -24,10 +25,12 @@ Ensure fapolicyd exception-file exists:
 {%- for fileType in awscli_filetypes %}
 Exempt AWS CLI v2 From fapolicyd ({{ fileType }}):
   file.replace:
-    - name: '/etc/fapolicyd/rules.d/30-aws.rules'
+    - name: '{{ exemptionFile }}'
     - append_if_not_found: True
     - onchanges_in:
       - cmd: 'Reload fapolicyd config'
+    - onlyif:
+      - '[[ -e {{ exemptionFile }} ]]'
```
Comment (lorengordon) [inline ash-linux/el8/VendorSTIG/aws_cli_v2.sls:33]: Looks like [`file.replace`](https://docs.saltproject.io/en/latest/ref/states/all/salt.states.file.html#salt.states.file.replace) can handle this natively...

```suggestion
    - ignore_if_missing: True
```

Comment (ferricoxide) [reply ash-linux/el8/VendorSTIG/aws_cli_v2.sls:33]: Ok. For some reason, it was in the back of my mind that it would create a null-file if that was set.

Comment (lorengordon) [reply ash-linux/el8/VendorSTIG/aws_cli_v2.sls]: I mean, I don't have everything setup to test. I'm just reviewing against the docs real quick. So give it a roll!

Comment (ferricoxide) [reply ash-linux/el8/VendorSTIG/aws_cli_v2.sls]: Yeah. Running it with the update, nao.

Comment (ferricoxide) [reply ash-linux/el8/VendorSTIG/aws_cli_v2.sls]: Ok, so, that worked as well.

**Review by lorengordon — APPROVED** [review]

**Pull request merged**